src/components/Minecraft/Create/FormCreation.tsx

```diff
@@ -0,0 +1,71 @@
+import { HandlerGameType } from 'bindings/HandlerGameType';
+import ComboField from 'components/Atoms/Form/ComboField';
+import InputField from 'components/Atoms/Form/InputField';
+import RadioField from 'components/Atoms/Form/RadioField';
+import { SectionManifest, SectionManifestValue, SettingManifest } from './form';
+import InputBox from 'components/Atoms/Config/InputBox';
+export const createForm = (section: SectionManifest) => {
+  const createField = (setting: SettingManifest) => {
+    console.log(setting);
+    switch (setting.value_type.type) {
+      case 'String':
+        return (
+          <InputField
+            type="text"
+            name={`${setting.setting_id}.value`}
+            label={setting.name ?? ''}
+          />
+        );
+      case 'Integer':
+      case 'UnsignedInteger':
+      case 'Float':
+        return (
+          <InputField
+            type="number"
+            name={`${setting.setting_id}.value`}
+            label={setting.name ?? ''}
+          />
+        );
+      case 'Boolean':
+        console.log(`${setting.setting_id}.value`);
+        return (
+          <RadioField
+            name={`${setting.setting_id}.value`}
+            label={setting.name ?? ''}
+            options={['false', 'true']}
+          />
+        );
+      case 'Enum':
+        return (
+          <ComboField
+            name={`${setting.setting_id}.value`}
+            label={setting.name ?? ''}
+            options={setting.value_type.options ?? []}
+            filterOptions={(query, options) => {
+              return query === ''
+                ? options
+                : options.filter((option) =>
+                    option.toLowerCase().startsWith(query.toLowerCase())
+                  );
+            }}
+          />
+        );
+    }
```

Comment:
It might be better to throw an error/show a toast here, otherwise if type doesn't match an option might just silently not show up